### DISCUSSION TITLE
phantom transaction

### DISCUSSION THREAD

**Discussion by karado58** [btcpayserver/btcpayserver#4037] [Technical Support]
I now have a phantom transaction in my BTC Wallet that don't belong to me!! I tried to "Scan" wallet, but said feature is disabled, and that I'm not the admin!! How can I refresh/update my BTC Wallet, please? 
Thanks.

![rescan2](https://user-images.githubusercontent.com/38022838/184549108-dc1279c3-c378-49ac-9f1c-8116c69e79c1.png)

**Comment by Kukks** [comment]
You need to provide more information. How did you configure the wallet inside the store in btcpay? Did you generate it or import it? Are you using a third party btcpay server?  How do you know it does not belong to you? Do you see any invoices inside your btcpay store?

**Reply by karado58** [reply]
Can I just give you my login credentials to BTCPay server please, and you can see for yourself?

**Reply by karado58** [reply]
Please give me a private way to send my login credentials? thanks
